browser.js

```diff
@@ -0,0 +1,73 @@
+var createHmac = require('create-hmac')
+
+exports.pbkdf2 = pbkdf2;
+function pbkdf2(password, salt, iterations, keylen, digest, callback) {
```

Comment:
in case I need to call the function it will be a deceleration that's hoisted, mainly it's just my habit, I can change it.